hazelcast-sql-core/src/test/java/com/hazelcast/sql/SqlOrderByTest.java

```diff
@@ -16,11 +16,7 @@
 
 package com.hazelcast.sql;
 
-import com.hazelcast.config.Config;
-import com.hazelcast.config.InMemoryFormat;
-import com.hazelcast.config.IndexConfig;
-import com.hazelcast.config.IndexType;
-import com.hazelcast.config.MapConfig;
+import com.hazelcast.config.*;
```

Comment:
Our checkstyle doesn't pass this with an error. `'.*' form of import should be avoided`

Reply:
My bad, that's just IntelliJ Idea's 'Optimize Imports' feature.
Fixed in [12c9391](https://github.com/hazelcast/hazelcast/pull/18113/commits/12c93911eb1d07f2742fe8a48110a0a56aa73ca5)